recipes/mockextras/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = "mockextras" %}
+{% set version = "1.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 09b77b91a9bffd475eaaeb631628bdfd9a7e3eed31d235290614bb38c7979692
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pytest
+    - coveralls
+    - pytest-cov
+    - coverage
+    - mock >=0.8.0  # [py<33]
```

Comment:
Do you actually need Python 2.7?  It would be much bettrer if this could be noarch: python

Reply:
Yes, this is a dependency for the `arctic` library (https://pypi.org/project/arctic/), which still supports `python 2.7`. In order to release a healthy conda package for arctic we need this package to be compatible with `python 2.7` also.